src/components/TodoList/TodoList.jsx

```diff
@@ -1,4 +1,8 @@
-// Add the required props
-export const TodoList = () => (
-  <>TodoList markup</>
+import './TodoList.scss';
+import { TodoInfo } from '../TodoInfo';
+
+export const TodoList = ({ todos }) => (
+  <section className="TodoList">
+    {todos.map(todo => <TodoInfo todo={todo} key={todo.id} />)}
```

Comment:
```suggestion
    {todos.map(todo => (
      <TodoInfo todo={todo} key={todo.id} />
    ))}
```
I think it's better to improve the readability